proxy/src/main/java/com/velocitypowered/proxy/event/VelocityEventManager.java

```diff
@@ -426,12 +425,14 @@ public void fireAndForget(final Event event) {
 
   private <E extends Event> void fire(final @Nullable CompletableFuture<E> future,
```

Comment:
`<E>` here as well

Reply:
Since *firing events* does have the restriction that the event fired extend `Event`, using `<E extends Event>` here is accurate. `fire(E)` has the same generic requirement.

If you'd like to remove `Event` entirely, that's possible too, although not the goal of this PR.